timesketch/lib/analyzers/yetiindicators.py

```diff
@@ -30,23 +32,32 @@ def __init__(self, index_name, sketch_id, timeline_id=None):
         """
         super().__init__(index_name, sketch_id, timeline_id=timeline_id)
         self.intel = {}
-        self.yeti_api_root = current_app.config.get("YETI_API_ROOT")
-        self.yeti_web_root = current_app.config.get("YETI_API_ROOT")
-        self.yeti_web_root.replace("/api", "")
+        root = current_app.config.get("YETI_API_ROOT")
+        if root.endswith("/"):
```

Comment:
Isn't this handled at the Yeti web layer?

Reply:
Yes, but it's to avoid sending queries like `/api/v2//blah`. It also depends on how people have their webserver setup, I suppose.